src/signing/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
 
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Project="..\Directory.Build.props" />
```

Comment:
Why is the standard import lookup changed?

Reply:
I don't really understand what you mean here? I just simplified the syntax.

Reply:
This pattern reduces maintainability:
* It makes it possible to accidentally skip over a dir.props if you add a new dir.props and forget to update all instances of this pattern in the child dirs.
* It increases the number of changes in the resulting PR even if you do manage to replace everything correctly.

Reply:
I reverted all these changes as they are just noise. Thanks for sharing your opinion.